### PR TITLE
Remove not required PackageReferences

### PR DESCRIPTION
**Pull request by pascalberger**
Remove not required PackageReferences

### CODE REVIEW

File: src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj [hunk 1]
```diff
@@ -27,9 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
-    <PackageReference Include="Cake.Core" Version="1.0.0" />
     <PackageReference Include="Cake.Testing" Version="1.0.0" />
-    <PackageReference Include="Cake.Issues" Version="0.9.0" />
```
Comment (jokay) [inline src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj:32]: I think this should not be part of this PR (version will be updated via https://github.com/cake-contrib/Cake.Issues.EsLint/pull/156).

Comment (pascalberger) [reply src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj]: `Cake.Issues` reference will be removed, not added or updated

**Review by jokay — APPROVED** [review]
